trade-finance-manager-ui/templates/case/cancellation/bank-request-date.njk

```diff
@@ -100,7 +100,7 @@
             text: "Continue",
             attributes: { "data-cy": "continue-button" }
           }) }}
-          <a class="govuk-link" href="/case/{{ dealId }}/deal" data-cy="cancel-link">Cancel</a>
+          <a class="govuk-link" href="cancel" data-cy="cancel-link">Cancel</a>
```

Comment:
Shouldn't this be `/cancel` or similar?

Reply:
This is working as shorthand for `/case/${dealId}/cancellation/cancel`. I'm never quite sure how links work in HTML but this seems to be replacing the `bank-request-date` at the end of the URL with `cancel`

Reply:
I think a failsafe is to have a backslash - can we add this in please? Should still pass

This might be some very oldschool thinking from me 🙈 but regardless it feels odd to _not_ have a backslash

Reply:
Just played around with this & the `cancel` will break if the URL has a `/` on the end! Great spot, will replace it with the full `/case/{{ dealId }}/cancellation/cancel`

Reply:
Ah ha, it's been a while for me, but now I remember this is why 😄 Great spot/testing 👍